Dockerfile

```
@@ -4,7 +4,13 @@ ARG IMG_TAG=latest
 FROM golang:1.21-alpine AS gaiad-builder
```

[We currently need a build system with Alpine < 3.19](https://github.com/CosmWasm/wasmvm/issues/523). This seems to do the job for most teams:

```suggestion
FROM golang:1.21-alpine3.18 AS gaiad-builder
```